src/Filters/Block.php

```diff
@@ -964,4 +967,69 @@ private function preparePriceSpecifications()
             'currencySymbol' => $currency->sign,
         ];
     }
+
+    /**
+     * Get the reviews filter block
+     *
+     * @param array $filter
+     * @param array $selectedFilters
+     *
+     * @return array
+     */
+    private function getReviweBlock($filter, $selectedFilters)
+    {
+        $values = [
+            '4' => ['name'=>"4"],
+            '3' => ['name'=>"3"],
+            '2' =>  ['name'=>"2"],
+            '1' => ['name'=>"1"],
+        ];
+
+        $query = 'SELECT  count(g.grade) as count ,g.grade FROM (SELECT t.id_product, case 
+when t.avg_grade between 1 and 1.99 then "1"
+when t.avg_grade between 2 and 2.99 then "2"
+when t.avg_grade between 3 and 3.99 then "3"
+when t.avg_grade between 4 and 5 then "4"
+end as grade
+ FROM (SELECT avg(h.grade) as avg_grade,
+ h.id_product FROM 
+ (select pc.id_product, pc.grade from ' . _DB_PREFIX_ . 'product_comment as pc
+inner join (SELECT * FROM ' . _DB_PREFIX_ . 'category_product where id_category = '.$_GET['id_category'].') cp
```

Comment:
Sql injection here, you should use `$idParent` defined in line 97

Reply:
will add.

Reply:
done